scarpe-components/Gemfile

```diff
@@ -8,11 +8,14 @@ gem "lacci", path: "../lacci"
 
 gem "rake", "~> 13.0"
 
-gem "nokogiri"
+#gem "nokogiri"
```

Comment:
Is nokogiri gone temporarily? Or do we no longer need it?

Reply:
Not currently used in the scarpe-components tests. It could be in future.